hw/ip/pwrmgr/data/pwrmgr_testplan.hjson

```diff
@@ -11,22 +11,225 @@
     {
       name: smoke
       desc: '''
-            Smoke test accessing a major datapath within the pwrmgr.
+            Smoke test exercising the pwrmgr state transitions.
```

Comment:
one more thing i forgot, the pwrmgr has a rom check for power up, so it would be good to check that too. (but this is probably pretty basic)

Reply:
Well, apparently if rom_ctrl_good is not set the fast fsm gets stuck in FastPwrStateRomCheck.

Reply:
yeah.. i'm trying to think if there's any other ways to check / stress this. 
the rom_ctrl indication really shouldn't change in general.. but I suppose it would be good have a test that makes sure once pwrmgr is past that point, random toggling of `rom_ctrl` should not make any difference until the next low power exit / reset.

Not a terribly important test i guess.

Reply:
I agree...